content/source/layouts/enterprise2.erb

```diff
@@ -213,6 +213,9 @@
           <li<%= sidebar_current("docs-enterprise2-private-config") %>>
             <a href="/docs/enterprise/private/config.html">Configuration</a>
           </li>
+          <li<%= sidebar_current("docs-enterprise2-private-vault") %>>
+            <a href="/docs/enterprise/private/vault.html">Customer Vault (Installer Beta)</a>
```

Comment:
thoughts on making these a sub-category under the Installation (Installer Beta) header rather than it's own section?

Reply:
Sure. I didn't do that because it appears that the sub-sections are hidden by default?

Reply:
Can I recommend: "External Vault" instead of "Customer Vault?" Customer is one of those relative terms where it can be hard to tell when You're It.